data-prepper-plugins/otel-trace-source/src/main/java/com/amazon/dataprepper/plugins/source/oteltrace/OTelTraceSource.java

```diff
@@ -115,7 +115,8 @@ public void start(Buffer<Record<Object>> buffer) {
 
             final Optional<Function<? super HttpService, ? extends HttpService>> optionalHttpAuthenticationService =
                     authenticationProvider.getHttpAuthenticationService();
-            optionalHttpAuthenticationService.ifPresent(sb::decorator);
+            optionalHttpAuthenticationService.ifPresent(httpAuthenticationService ->
+                    sb.decorator("regex:^/(?!health$).*$", httpAuthenticationService));
```

Comment:
Consider moving the regex pattern to static constant. Can we create a pre-complied constant pattern ?

Reply:
I moved it to a constant. The function only takes in a string, not a Pattern